events/README.md

```diff
@@ -22,9 +22,9 @@ stato concesso per una fruizione pubblica dai rispettivi autori.
 
 # 22 Set 2023 | Under the hood at Klarna: Managing Digital Gift Cards and Leveraging AI | [The SPACE Coworking](https://www.thespacecoworking.website/)
 
-| Titolo | Autore | Titolo | Autore | Titolo | Autore | Titolo | Autore | Asset |
-| --- | --- | --- | --- | --- | --- | --- | --- | --- |
-| LiT - Welcome! | [LiT](https://www.linkedin.com/company/latina-in-tech/) | Welcome to Klarna | [Antonio Trapani](https://www.linkedin.com/in/antoniotrapani/) | Gift Cards Team | [Andrea Coluzzi](https://www.linkedin.com/in/andrea-coluzzi/) | The New Gold Rush | [Antonio Trapani](https://www.linkedin.com/in/antoniotrapani/) | [PDF](https://drive.google.com/file/d/1ifyBJwKRySKLRShrvcMsej9mOHtO2Hf4/view?usp=drive_link) |
+| Titolo | Autore | Asset |
+| --- | --- | --- |
+| LiT - Welcome! <br/> Welcome to Klarna <br/>Gift Cards Team <br/> The New Gold Rush | [LiT](https://www.linkedin.com/company/latina-in-tech/) </br> [Antonio Trapani](https://www.linkedin.com/in/antoniotrapani/) <br/> [Andrea Coluzzi](https://www.linkedin.com/in/andrea-coluzzi/) <br/> [Antonio Trapani](https://www.linkedin.com/in/antoniotrapani/) | [PDF](https://drive.google.com/file/d/1ifyBJwKRySKLRShrvcMsej9mOHtO2Hf4/view?usp=drive_link) |
```

Comment:
ho messo la tabella su un blocco unico avendo tutti autori nello stesso PDF anziché ripeterli N volte

Reply:
Pre:
<img width="1054" alt="Screenshot 2024-06-12 at 14 11 44" src="https://github.com/latina-in-tech/lit-assets/assets/54796163/0e5a2aa5-92df-4c55-ac9c-3bfc95b9b81e">


Post:
<img width="342" alt="Screenshot 2024-06-12 at 14 11 28" src="https://github.com/latina-in-tech/lit-assets/assets/54796163/c73c5774-915e-4c62-b51c-221515cd30a1">

Reply:
@coluzziandrea